TASVideos/Pages/Activity/Publishers.cshtml

```diff
@@ -2,8 +2,30 @@
 @{ ViewData["Title"] = $"Publisher Activity for {Model.UserName}"; }
 @model PublishersModel
 
+@{
+	var counts = new Dictionary<string, int>();
```

Comment:
the combination of 
.GroupBy()
.ToDictionary()

Will simplify this code a good bit